File: flutter_covid/lib/Models/case.dart
class Case {
  String name;
  String birthdate;
  String risk;
  String sex;
  String email;
  String phone;
  String address;
  String clasi;

  Case(this.name, this.birthdate, this.risk, this.sex, this.email, this.phone,
      this.address, this.clasi);

  factory Case.fromJson(dynamic json) {
    return Case(
      json['name'] as String,
      json['birthdate'] as String,
      json['risk'] as String,
      json['sex'] as String,
      json['email'] as String,
      json['phone'] as String,
      json['address'] as String,
      json['clasi'] as String,
    );
  }
}

File: Backend/src/models/case.ts
import  {model, Schema, Document} from "mongoose";

export interface ICase extends Document {
    name: string;
    birthdate: string;
    date: string;
    risk: string;
    sex: string;
    email: string;
    phone: string;
    address: string;
    class: string;
}

const caseSchema = new Schema({
    name: {type: String},
    birthdate: {type: String},
    date: {type: String},
    risk: {type: String},
    sex: {type: String},
    email: {type: String},
    phone: {type: String},
    address: {type: String},
    class: {type: String}
})

export default model<ICase>("Case", caseSchema, "cases");
